app.py

```diff
@@ -13,13 +13,13 @@
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-from aws_cdk import core as cdk
+from aws_cdk import App
```

Comment:
I'd prefer to be explicit about `App`'s module, since this is quite a generic word:
```suggestion
import aws_cdk as cdk
```